assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/properties/components/table/table.tsx

```diff
@@ -125,13 +125,15 @@ export const Table = ({ propertiesTableTab }: ITableProps): React.JSX.Element =>
               )
             }
 
-            <Button
-              icon={ <Icon name="trash" /> }
-              onClick={ () => {
-                removeProperty(info.row.original)
-              } }
-              type="link"
-            />
+            {propertiesTableTab === 'own' && (
+              <Button
```

Comment:
Now where I see it, we could already implement the new Button with text and icon component here.

Reply:
Ahhh, nevermind. :D  

I will refine the api of the button a bit more beforehand. ;D
So far it's not that compatible to ant design.